document/cloud-client/batch_parse_table.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START documentai_batch_parse_table]
+from google.cloud import documentai_v1beta2 as documentai
+from google.cloud import storage
+import re
+
+
+def batch_parse_table(
+        project_id='YOUR_PROJECT_ID',
+        input_uri='gs://cloud-samples-data/documentai/form.pdf',
+        destination_uri='DESTINATION_URI'):
+    """Parse a form"""
+
+    client = documentai.DocumentUnderstandingServiceClient()
+
+    gcs_source = documentai.types.GcsSource(uri=input_uri)
+
+    # mime_type can be application/pdf, image/tiff,
+    # and image/gif, or application/json
+    input_config = documentai.types.InputConfig(
+        gcs_source=gcs_source, mime_type='application/pdf')
+
+    # where to write results
+    output_config = documentai.types.OutputConfig(
+        gcs_destination=documentai.types.GcsDestination(
+            uri=destination_uri),
+        pages_per_shard=1  # Map one doc page to one output page
+    )
+
+    # Improve table parsing results by providing bounding boxes
+    # specifying where the box appears in the document (optional)
+    table_bound_hints = [
+        documentai.types.TableBoundHint(
+            page_number=1,
+            bounding_box=documentai.types.BoundingPoly(
+                normalized_vertices=[
+                    documentai.types.geometry.NormalizedVertex(
+                        x=0,
+                        y=0
+                    ),
+                    documentai.types.geometry.NormalizedVertex(
+                        x=1,
+                        y=0
+                    ),
+                    documentai.types.geometry.NormalizedVertex(
+                        x=1,
+                        y=1
+                    ),
+                    documentai.types.geometry.NormalizedVertex(
+                        x=0,
+                        y=1
+                    )
```

Comment:
Does ordering matter?
Top Left --> Top Right --> Bottom Right --> Bottom Left

Nit: maybe add a comment for that? idk

Reply:
Done.

We should probably improve these comments post launch.